_src/SetupHelper.php

```diff
@@ -24,6 +24,20 @@ public static function getLowerCase($string)
         return str_replace(' ', '-', mb_strtolower($string));
     }
 
+    /**
+     * Remove quotes and/or double-quotes enclosing the string.
+     *
+     * @param string $string String to be sanitized.
+     *
+     * @return string Converted string.
+     */
+    public static function getUnquoted($string)
+    {
+        $string = trim($string,"'");
+        
+        return trim($string, '"');
```

Comment:
`trim()` allows more than one character to be defined for trimming so you can combine the two function calls into one:

```suggestion
        return trim($string,'\'"');
```